massdriver.yaml

```
@@ -317,6 +317,18 @@ params:
             - "14.4"
             - "14.5"
             - "14.6"
+            - "14.7"
+            - "14.8"
+            - "14.9"
+            - "14.10"
+            - "14.11"
+            - "15.2"
+            - "15.3"
+            - "15.4"
+            - "15.5"
+            - "15.6"
+            - "16.1"
+            - "16.2"
           default: "14.6"
```

```suggestion
          default: "16.2"
```
Let's keep the default set to the most recent available version